tests/framework/Optimizers/SimulatedAnnealing/BoltzmanEggHolder/test_simulatedAnnealing_eggHold.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="all">
+  <TestInfo>
+    <name>framework/Optimizers.SimulatedAnnealing</name>
```

Comment:
this is not the right name... 
it should be ```framework/Optimizers/SimulatedAnnealing.BoltzmanEggHolder```

this comment is valid for all the other tests. Please fix all of them

Reply:
I made it: `framework/Optimizers/SimulatedAnnealing.Boltzmann.EggHolder` or should it be consistent with the folder where it lives?

Reply:
It should be the path that the `run_tests` calls it on, which is `/path/to/folder.testName`